configure.ac

```diff
@@ -116,6 +116,19 @@ AC_ARG_ENABLE([gles],
     AC_DEFINE([USE_GLES], [1], [Define USE_GLES])
 ])
 
+dnl LLVM
+dnl unfortuately AX_LLVM macro seems to be out of date, so we're going to rely on the user to make sure LLVM is installed correctly
+AC_ARG_ENABLE([llvm],
+  AS_HELP_STRING([--enable-llvm],[Support for JIT using LLVM]),
+  [], [enable_llvm=no])
+AS_IF([test x"$enable_llvm" = "xyes"], [
+    AC_DEFINE([HAVE_LLVM], [1], [Define HAVE_LLVM])
+    CFLAGS="$CFLAGS -I$(llvm-config --includedir)"
```

Comment:
i think it's better to set LLVM_CXXFLAGS and LLVM_LDFLAGS and add those to compilation of libprojectM instead of mangling these flags globally

Reply:
Please update as you think is best.  I'm in over my head with the build.